assets/js/script.js

```diff
@@ -0,0 +1,51 @@
+$(document).ready(function() {
```

Comment:
Future note: maybe rewrite this without the JQuery? Sounds like the latest bootstrap dropped the JQuery dependency 👍

Reply:
Yeah, although I don't think it greatly affects the load time.

Reply:
I'd like to get rid of AOS (scroll to library) if possible